# Assignment 1

Mr.Tanakrit Pichaitam  6137760SIMB/M

## Questions

1.What type of study design was used in this study?

```
Case-control comparisons study
```

2.What is the inclusion and exclusion criteria of the study population? Choose two diseases mentioned in the paper and provide the detail of inclusion & exclusion criteria.

```
Control groups recruit 2,000 samples were genotyped for each. The control samples came from two sources: half from the 1958 Birth Cohort and the remainder from a new UK Blood Service sample. That use two control groups which have noting that DNA sample processing differed between these groups. And used ‘hyper controls’ (selection of control individuals from the lower extremity of the relevant trait distribution) to focusing on one disease which weigh against the additional costs associated with the need to phenotype and genotype each control sample. This study provided individuals with non-European ancestry are excluded, the extent of population stratification in the British population is generally modest. QC/QA measures included sample call rate, overall heterozygosity and evidence of non-European ancestry (809 samples excluded; 16,179 retained for analysis). The population were excluded 809 samples after checks for contamination, false identity, non-Caucasian ancestry and relatedness.

coronary artery disease (CAD) cases which are myocardial infarction or coronary revascularization (coronary artery bypass surgery or percutaneous coronary angioplasty) before their 66th birthday.

type 1 diabetes (T1D) cases recruit from pediatric and adult diabetes clinics at 150 National Health Service hospitals across mainland UK. The total case data set around 8,000 from the WTCCC cases. all cases to have an age of diagnosis below 17 years old and insulin dependence since diagnosis with a minimum period of at least 6 months. The cases which are suffering from rare monogenic disorders including, maturity onset diabetes of the young (MODY), and latterly permanent neonatal diabetes (PNDM) are exclude.
```

3.Was the age distribution of each disease group comparable to the age distribution of the control groups?
3.1. What statistical test can you use to compare the age distribution between the two groups?

```
This study has no statistical test age distribution between the two groups. For my opinion, this study can be provided discriminant analysis though the age distribution.
```

3.2. Which disease groups had significantly different age of distribution than controls? Also provide the statistical evidence for the comparison.

```
1958 Birth Cohort Controls (58BC)
The 1958 Birth Cohort (the National Child Development Study) includes all births in England, Wales and Scotland, during one week in 1958. From an original sample of over 17,000 births, survivors were followed up at ages 7, 11, 16, 23, 33 and 42 years old. In a biomedical examination at 44-45 years old

UK Blood Services Controls (UKBS)
Which set up from a UK national repository of anonymized samples of DNA and viable mononuclear cells from 3,622 consenting blood donors, age range 18–69 years old.
```

3.3. Pick 1 disease with different age distribution between patients and control group. Discuss how the different in age distribution might affect the results of genetic association.

```
Type 1 diabetes (T1D) which onset usually in childhood which are significant. required all cases to have an age of diagnosis below 17 years old and insulin dependence since diagnosis (with a minimum period of at least 6 months)
```

4.Besides age, were there any other confounding factor(s) in the design of this study?

```
During the geographical and population selection can be exclude confounding sample out.
```

5.What is the approach that we might be able to control those confounding factors?

```
By Randomization, Restriction and Matching. In this study the statistical methods to adjust for potentially confounding effects rely on geographical variation and population structure. ```

6.For the strongest association signal for coronary artery disease,

6.1. What is the location of this SNP?

```
SNP in MTHFD1L
```
   
6.2. Is there a gene nearby?
   
```
chromosome 9p21.3 genes for two cyclin dependent kinase inhibitors consist of CDKN2A and CDKN2B
```

6.3. Was the SNP known or has any biological mechanism to affect the gene function?

```
encodes methylenetetrahydrofolate dehydrogenase (NADP+-dependent) 1-like, the mitochondrial isozyme of C1-tetrahydrofolate (THF) synthase
```

6.4. Is it biologically plausible that we found this SNP associated with CAD?

```
The possibility of a link between variants in MTHFD1L and CAD risk that MTHFD1L activity contributes to plasma homocysteine and defects in the MTHFD1L pathway may increase plasma homocysteine level
```
