SIRE 506 Design and Analysis in Genetic Studies of Human Diseases
===============

## Academic Year: 2020

## Date:  Jan 4 – April 30, 2021
-   [Course Website](https://si-medbif.github.io/SIRE506/)
-   [Github Repository](https://github.com/si-medbif/SIRE506)
-   [Handouts]()

## Schedule

| DATE | TIME | ROOM | Evaluation | Topic | FACULTY STAFF | Coursera Supplements |
|------|------|------|------------|-------|---------------|----------------------|
| 1/4/2021 | 9:00-12:00 | SIMR Fl.11 | A+E | Complex Genetic Diseases in Population | Bhoom  Suktitipat | [The Data Scientist's Toolbox Week 2, 4](https://www.coursera.org/learn/data-scientists-tools?specialization=data-science-foundations-r) |
| 1/11/2021 | 9:00-12:00 | SIMR Fl.11 | A | Exploratory Data Analysis | Bhoom  Suktitipat | [Exploratory Data Analysis](https://www.coursera.org/learn/exploratory-data-analysis?specialization=jhu-data-science) |
| 1/18/2021 | 9:00-12:00 | SIMR Fl.11 | A+E | Study designs in epidemiology and genetics I | Bhoom  Suktitipat | [Study Design in Epidemiology - Week 1-2](https://www.coursera.org/learn/study-designs-epidemiology) |
| 1/25/2021 | 9:00-12:00 | SIMR Fl.11 | A+E | Study designs in epidemiology and genetics II | Bhoom  Suktitipat | [Study Design in Epidemiology - Week 3-4](https://www.coursera.org/learn/study-designs-epidemiology)|
| 2/1/2021 | 9:00-12:00 | SIMR Fl.11 | A | Statistical Inference I | Bhoom  Suktitipat | [Statistical Inference - Week 1-2](https://www.coursera.org/learn/statistical-inference) |
| 2/8/2021 | 9:00-12:00 | SIMR Fl.11 | A | Statistical Inference II | Bhoom  Suktitipat | [Statistical Inference - Week 3-4](https://www.coursera.org/learn/statistical-inference) |
| 2/15/2021 | 9:00-12:00 | SIMR Fl.11 | A | Statistics in Genomics | Bhoom  Suktitipat | [Statistics for Genomic Data Science - Week 1 & 2](https://www.coursera.org/learn/statistical-genomics) |
| 2/22/2021 | 9:00-12:00 | SIMR Fl.11 | A+E | Designs and analysis in quantitative trait genetics studies | Pimphen Charoen | [Statistics for Genomic Data Science - Week 2](https://www.coursera.org/learn/statistical-genomics) |
| 3/1/2021 | 9:00-12:00 | SIMR Fl.11 | A+E | Designs and analysis of binary traits in genetics studies | Pimphen Charoen | [Statistics for Genomic Data Science - Week 3](https://www.coursera.org/learn/statistical-genomics) |
| 3/8/2021 | 9:00-12:00 | SIMR Fl.11 | A+E | Overview of GWAS/WGS, eQTL, GSEA, RNASeq, ChipSeq, Methylation | Bhoom  Suktitipat | [Statistics for Genomic Data Science - Week 4](https://www.coursera.org/learn/statistical-genomics) |
| 3/15/2021 | 9:00-12:00 | SIMR Fl.11 | A | Eigenstrat & PCA for Population Substructure | Jakris Euahsunthornwattana |  |
| 3/22/2021 | 9:00-12:00 | SIMR Fl.11 | A+E | Mendelian randomization & meta-analysis | Pimphen Charoen |  |
| 3/29/2021 | 9:00-12:00 | SIMR Fl.11 | A | Differential Expression Analysis and GSEA & Pathway analysis | Varodom Charoensawan |  |
| 4/5/2021 | 9:00-12:00 | SIMR Fl.11 | A+E | Genetic studies using next-generation sequencing technology | Harald Grove |  |
| 4/12/2021 | 9:00-12:00 | SIMR Fl.11 | A | Precision Medicine and Public Health | Bhoom Suktitipat | [Case studies in Personalized Medicine -- Week 3](https://www.coursera.org/learn/personalizedmed) |
| 4/19/2021 | 9:00-12:00 | SIMR Fl.11 |   | Final exam | Faculty |  |

## Course Learning Objectives
1.	Differentiate mendelian and complex genetic diseases, and explain the factors influencing the disease expression, such as heritability, penetrance, variable expressivity, genetic heterogeneity, phenocopy, pleiotropy. 
2.	Can perform exploratory data analysis and create a descriptive summary statistics report similar to the “Table 1 summary of study population demographics” using R. 
3.	Can systematically describe the various methodological approaches in epidemiological research. 
  -	Can differentiate the descriptive from analytic study design, experimental vs observational, explain the different between cross-sectional study and case-control study, and highlight the strengths and weaknesses through real-life examples. 
  -	Can explain what ecological study design is, and when ecological fallacy can happen. And, properly choose the optimal design to answer the questions in genetic research. 
  -	Can design a genetic association study to investigate common genetic diseases, including creating a case record form to collect data. Drawing an example from genome-wide association study of height
4.	Can explain the advantage of cohort studies over case-control studies, and give some examples of how cohort study design is used in genetic studies. Drawing examples from gene expression study to identify SPM in HNSCC and the three studies in Manolio 2009 (https://www.nature.com/articles/ng0109-5).
5.	Can perform basic calculation of common descriptive statistic estimates e.g. mean, standard deviation, variance, standard error. Can calculate the posterior probability using the Bayes Theorem. 
6.	Can choose the appropriate statistical tests for quantitative variables and categorical variables, set up the hypothesis testing process, and interpret the statistical significance of the results from the p-value and 95% confidence interval. 
7.	Can perform clustering analysis and interpret the results from Heirachical Clustering and K-Mean methods
  -	Can propely recode the data for additive, dominant, recessive, and co-dominant genetic model; interpret the results from each model.
8.	Can perform linear regression analysis and interpret the results of a multivariate linear regression analysis
  -	Can analyze and interpret the results from the linear regression analysis from the study of quantitative phenotypes
9.	Can perform logistic regression analysis and interpret the results of a multivariate logistic regression analysis
  -	Can analyze and interpret the results from the logistic regression analysis from a case-control study of binary phenotypes
10.	Can describe the objectives of common study designs used in genetic studies, namely GSEA, RNASeq, ChipSeq, Methylation, GWAS/WGS, eQTL.
  -	Can perform quality control check of high-throughput SNP genotyping and create a report
  -	Can perform GWAS and write a report on the results.
11.	Can perform population stratification analysis using Eigenstrat and PCA, and can adjust for population substruction in genetic association analysis
12.	Can explain the objectives of mendelian randomization and meta-analysis are used in genetic studies
13.	Can perform GSEA and pathway analysis
14.	Can prioritize variants for report according to ACMG recommendation
15.	Can explain how the finding from genetic studies are used in precision medicine and public health settings

## Scoring


* Attendance: 10%
* Homework: 60%
* Final Exam: 30%

## Grading


* A > 90%
* B+ [80-90)
* B [70-80)
* C < 70%

## Course coordinator:

Asst. Prof. Dr. Bhoom Suktitipat, MD, PhD  
Graduate Program in Medical Bioinformatics  
Department of Biochemistry, Faculty of Medicine Siriraj Hospital &  
Integrative Computational BioScience Center, Mahidol University

## Instructors
* Dr. Harald Grove, PhD (Faculty of Medicine Siriraj Hospital, Mahidol University)
* Dr. Jakris Eu-ahsunthornwattana, MD, PhD (Faculty of Medicine Ramathibodi Hospital, Mahidol University)
* Dr. Pimphen Charoen, PhD (Faculty of Tropical Medicine, Mahidol University)
* Assoc. Prof. Dr. Vardodom Charoensawan, PhD (Faculty of Science, Mahidol University)

## Reference Books
1. Laird NM, Lange C. The Fundamentals of Modern Statistical Genetics. Springer Science & Business Media; 2010. 226 p.
2. Thomas DC. Statistical methods in genetic epidemiology. Oxford University Press; 2004. 458 p.
3. Ziegler A, König IR. A statistical approach to genetic epidemiology: Concepts and Applications. John Wiley & Sons; 2010. 522 p.

## Handouts
Please see this [link]()
